app/index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Amar Muha</title>
+  <link rel="stylesheet" href="styles/style.css">
+</head>
+
+<body>
+  <div class="wrapper">
+    <section class="header-section">
+      <header class="main-header">
+        <div class="logo">
+          Amar
+        </div>
+        <div class="toggler" id="toggler"></div>
+        <!-- <ul class="nav" id="show">
+          <li class="nav-elements">Portfolio</li>
+          <li class="nav-elements">About</li>
+          <li class="nav-elements">Contact</li>
+        </ul> -->
```

Comment:
- [ ] [OPTIONAL] Please I suggest removing uncommented code from to make your project look clean and professional. 😃